client/src/transmitter/cast/webrtc.ts

```diff
@@ -0,0 +1,71 @@
+const log = (...data) => console.log('webrtc', ...data)
+
+async function iceHandshake(
+  streamKey: string,
+  localDesc: RTCSessionDescription
+) {
+  const qsKey = streamKey.indexOf('://') > 0 ? 'rtmp' : 'streamKey'
+  const answer = await fetch(
+    `https://webrtmp.justcast.it/webrtc/offer?${qsKey}=${streamKey}`,
+    {
+      method: 'POST',
+      body: JSON.stringify(localDesc),
+      headers: {
+        ['content-type']: 'application/json',
+      },
+    }
+  )
+  if (answer.status !== 200) {
+    throw new Error(`Error response from server: ${answer.status}`)
+  }
+  return (await answer.json()) as RTCSessionDescriptionInit
```

Comment:
NIT
```
const init = await answer.json() as RTCSessionDescriptionInit

return init
```

Reply:
Thx, went with
```
  const remoteDesc = await answer.json()
  return remoteDesc as RTCSessionDescriptionInit
```
otherwise our linter would still force the ugly parenthesis over the `await`. Use `remoteDesc` which is how we are calling it on the code that calls this, in contrast with the `localDesc`.